### DISCUSSION TITLE
[1.1.432] ControlNet Doesn't Filter Model & Preprocessor after choose control type

### DISCUSSION THREAD

**Discussion by pandoleya** [Mikubill/sd-webui-controlnet#2515] [Q&A] [answered]
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/13018639/a22c10d4-1584-4435-81c6-c40dc96216cf)

**Comment by huchenlei** [comment]
What are the steps to reproduce this issue?

You can try update to latest version, https://github.com/Mikubill/sd-webui-controlnet/pull/2512 might fixes this issue.

**Reply by huchenlei** [reply, marked as answer]
There is also a possibility that your A1111 version is too old that some A1111 elements expected by ControlNet do not exist so callbacks are not registered.

Please try update your A1111 to 1.7.0+ as well.

**Reply by pandoleya** [reply]
Thank you! Issue solved after getting A1111 up-to-date
